src/client/src/Finsemble/index.tsx

```diff
@@ -1,71 +1,44 @@
-import { BrowserRouter, Route, Switch } from "react-router-dom"
+import { BrowserRouter, Route, Routes } from "react-router-dom"
 
 import { Analytics } from "@/App/Analytics"
 import { LiveRates } from "@/App/LiveRates"
-import { TileView } from "@/App/LiveRates/selectedView"
-import { TornOutTile } from "@/App/LiveRates/Tile/TearOut/TornOutTile"
 import { FxTrades } from "@/App/Trades"
 import { DisconnectionOverlay } from "@/components/DisconnectionOverlay"
 import { DocTitle } from "@/components/DocTitle"
 import { BASE_PATH, ROUTES_CONFIG } from "@/constants"
+import { TileWrapper } from "@/Web"
 
 export const FinsembleApp = () => (
   <BrowserRouter basename={BASE_PATH}>
-    <Switch>
+    <Routes>
       <Route
         path={ROUTES_CONFIG.analytics}
-        render={() => (
+        element={
           <DocTitle title="Analytics">
             <Analytics hideIfMatches={null} />
             <DisconnectionOverlay />
           </DocTitle>
-        )}
+        }
       />
       <Route
         path={ROUTES_CONFIG.blotter}
-        render={() => (
+        element={
           <DocTitle title="Trades">
             <FxTrades />
             <DisconnectionOverlay />
           </DocTitle>
-        )}
+        }
       />
       <Route
         path={ROUTES_CONFIG.tiles}
-        render={() => (
+        element={
           <DocTitle title="Live Rates">
             <LiveRates />
             <DisconnectionOverlay />
           </DocTitle>
-        )}
+        }
       />
-      <Route
-        path={ROUTES_CONFIG.tile}
-        render={({
-          location: { search },
-          match: {
-            params: { symbol },
-          },
-        }) => {
-          const query = new URLSearchParams(search)
-          const view = query.has("tileView")
-            ? (query.get("tileView") as TileView)
-            : TileView.Analytics
-
-          return (
-            <>
-              {symbol && (
-                <TornOutTile
-                  symbol={symbol}
-                  view={view}
-                  supportsTearOut={false}
-                />
-              )}
-              <DisconnectionOverlay />
-            </>
-          )
-        }}
-      />
-    </Switch>
+      <Route path={ROUTES_CONFIG.tile} element={<TileWrapper />} />
```

Comment:
ah, nice .. but can we use common location, I don't want to import from Web in Finsemble or OpenFin